### PR TITLE
`main` merge into `feat/beta`

### PR DESCRIPTION
**Pull request by andreiblt1304**
This is a preliminary merge just to have all the logic from the latest release, this one will be followed by a merge of `feat/enshrine` that will have all the _enshrine_ logic implementation

### CODE REVIEW

File: enshrine-esdt-safe/src/to_sovereign/create_tx.rs [hunk 1]
```diff
@@ -94,14 +94,14 @@ pub trait CreateTxModule:
             return;
         }
 
-        let all_names = self.verify_items_signature(opt_signature, names);
-        self.banned_endpoint_names().extend(&all_names);
+        // let all_names = self.verify_items_signature(opt_signature, names);
```
Comment (mihaicalinluca) [inline enshrine-esdt-safe/src/to_sovereign/create_tx.rs:97]: commented code

File: enshrine-esdt-safe/src/to_sovereign/create_tx.rs [hunk 2]
```diff
@@ -111,8 +111,8 @@ pub trait CreateTxModule:
             return;
         }
 
-        let all_names = self.verify_items_signature(opt_signature, names);
-        self.remove_items(&mut self.banned_endpoint_names(), &all_names);
+        // let all_names = self.verify_bls_signature(opt_signature, names);
+        // self.remove_items(&mut self.banned_endpoint_names(), &all_names);
```
Comment (mihaicalinluca) [inline enshrine-esdt-safe/src/to_sovereign/create_tx.rs:115]: commented code

File: enshrine-esdt-safe/src/to_sovereign/create_tx.rs [hunk 3]
```diff
@@ -60,14 +60,14 @@ pub trait CreateTxModule:
             return;
         }
 
-        let all_tokens = self.verify_items_signature(opt_signature, tokens);
-        self.burn_tokens().extend(&all_tokens);
+        // let all_tokens = self.verify_bls_signature(opt_signature, tokens);
```
Comment (mihaicalinluca) [inline enshrine-esdt-safe/src/to_sovereign/create_tx.rs:63]: commented code

File: enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs [hunk 4]
```diff
@@ -287,18 +285,18 @@ pub trait TransferTokensModule:
...
-            OperationData {
-                op_nonce: tx_nonce,
-                op_sender: sc_address.clone(),
-                opt_transfer_data: None,
-            },
-        );
+        let _sc_address = self.blockchain().get_sc_address();
+        let _tx_nonce = self.get_and_save_next_tx_id();
+
+        // self.deposit_event(
```
Comment (mihaicalinluca) [inline enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs:291]: commented code

File: enshrine-esdt-safe/src/to_sovereign/create_tx.rs (hunk 2 above)
Comment (andreiblt1304) [reply enshrine-esdt-safe/src/to_sovereign/create_tx.rs:115]: this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced

File: enshrine-esdt-safe/src/to_sovereign/create_tx.rs (hunk 3 above)
Comment (andreiblt1304) [reply enshrine-esdt-safe/src/to_sovereign/create_tx.rs:63]: this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced

File: enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs (hunk 4 above)
Comment (andreiblt1304) [reply enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs:291]: this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced

**Review by mihaicalinluca — APPROVED** [review]

**Review by axenteoctavian — APPROVED** [review]

**Pull request merged**